packages/colors/lib/colors.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = colors;
```

Comment:
please don't use commonjs in 2019 😢

Reply:
`lerna create` did it 🙈this package is empty, because we will implement colors after switching to lerna.